packages/hub/config/structure.sql

```diff
@@ -1527,6 +1551,13 @@ ALTER TABLE ONLY public.wyre_prices
 CREATE INDEX jobs_priority_run_at_id_locked_at_without_failures_idx ON graphile_worker.jobs USING btree (priority, run_at, id, locked_at) WHERE (attempts < max_attempts);
 
 
+--
+-- Name: account_nonces_account_address_chain_id_unique_index; Type: INDEX; Schema: public; Owner: postgres
+--
+
+CREATE UNIQUE INDEX account_nonces_account_address_chain_id_unique_index ON public.account_nonces USING btree (account_address, chain_id);
```

Comment:
i think you actually have a multi-column uniqueness constraint--specifically the account_address, chain_id, nonce columns form a tuple that should be unique. it looks like the nonce column is missing from this constraint.